octobot_trading/exchanges/connectors/ccxt_exchange.py

```diff
@@ -284,11 +461,26 @@ async def get_recent_trades(self, symbol: str, limit: int = 50, **kwargs: dict)
     async def get_price_ticker(self, symbol: str, **kwargs: dict) -> typing.Optional[dict]:
         try:
             with self.error_describer():
-                return await self.client.fetch_ticker(symbol, params=kwargs)
+                ticker = await self.client.fetch_ticker(symbol, params=kwargs)
+                # some exchanges are missing values
+                if not ticker[ecoc.TIMESTAMP.value]:
```

Comment:
should we create a ticker parser for these lines ?

Reply:
yeah I created it